bugbug/repository.py

```diff
@@ -20,6 +25,20 @@
 BUG_PATTERN = re.compile('[\t ]*[Bb][Uu][Gg][\t ]*([0-9]+)')
 
 
+def hg_log(repo_dir):
+    hg = hglib.open(repo_dir)
+    args = cmdbuilder(b'log', template='{rev}\\0{node}\\0{tags}\\0{branch}\\0{author}\\0{desc}\\0{date}\\0')
+    x = hg.rawcommand(args)
+    out = x.split(b('\0'))[:-1]
```

Comment:
No need to use the `b` function, you can just hardcode the result of `b('\0')` (which is `b'\x00'`).